requirements.txt

```diff
@@ -1,6 +0,0 @@
-Django==1.10
```

Comment:
Hey arnold, can you please clarify on your reason for deleting the content in this file and not replacing it with updated versions of the dependancies. Thank you

Reply:
I actually do not remember deleting the contents, i might have done pip freeze < requirements.txt instead of pip freeze > requirements. This is now okay, kindly check again. Thanks for the observation.